scripts/monorepo/jest.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     '^.+\\.tsx?$': ['@swc/jest', {}],
   },
   coverageDirectory: './coverage',
+  testTimeout: 20000,
```

Comment:
this project uses nx graph for integration purposes - test timouts needs te be extended in order to not fail on "flaky" behaviours

Reply:
The `nx graph` is not utilized by Jest during test execution. Rather, its function is to identify which tasks should be executed for particular projects. Is that correct, or is there something I'm missing?

Reply:
one of public apis of this package `getDependencies`, that also has test coverage, is using nx graph (`createProjectGraphAsync`)
<img width="1142" alt="image" src="https://github.com/user-attachments/assets/d57c301b-e1c1-4683-8e96-cf5e893a845e">

meaning any package that uses nx graph api our our api that uses it and is part of tests needs to turn of deamon and increase testTimeout. there are ofc cases where we mock nxGraph because we are not interested in e2e/integration case scenario.

also see following:

https://github.com/microsoft/fluentui/pull/31949#discussion_r1691376861

https://github.com/microsoft/fluentui/commit/3da4204265901049d9ef777df15cb25b881a6284#diff-b6aa15e915a897fe80f71a79e9ec0ae4a5d06ec58a20ab47bb96bf213413fe2f

Reply:
this change should have been probably a separate PR, but it started to appear more often on repeated runs when enabling inferred task so I did the change as part of this PR.

Reply:
Thanks for the clarifications!